### PR TITLE
Update start button styles

### PR DESCRIPTION
**Pull request by Omelekhin**
Added Iset Sans font and updated text appearance.

### CODE REVIEW

File: view/header/index.css [hunk 1]
```diff
@@ -17,30 +17,31 @@
...
-  margin: 0 0 -.2em;
-  padding: 0.6em 0.5em;
-  color: black;
-  background: white;
+  padding: .5em 0;
+  color: #ffd400;
   text-decoration: none;
-  transition: .1s ease;
+  box-shadow: 0 1px 0 rgba(255,212,0,.5);
+  transition: all .05s ease;
```
Comment (sashachabin) [inline view/header/index.css]: 50ms?
is such a transition visible?)

maybe remove or set 100ms?

File: view/header/index.css [hunk 2]
```diff
@@ -17,30 +17,31 @@
...
+    width: 100%;
   }
 }
 
 .header__more {
   display: inline-block;
-  font-size: 5vw;
-  font-weight: 300;
-  line-height: 1.5;
+  font-size: .5vw !important;
```
Comment (sashachabin) [inline view/header/index.css]: Let's remove `!important`

File: index.html [hunk 3]
```diff
@@ -64,7 +64,7 @@
   </div>
 
   <main class="page">
-    <p style="page__text">
+    <p class="page__text">
```
Comment (sashachabin) [inline index.html]: 😅😅

Comment (sashachabin) [reply index.html]: thanks for fixing it!

**Comment by sashachabin** [comment]
It seems that the `preview` block has become cropped

image

UPD: this is an existing problem

File: view/header/index.css (hunk 2 above)
Comment (sashachabin) [reply view/header/index.css]: `.5vw` not readable at small devices

image

**Comment by sashachabin** [comment]
I think we should rename this to `lead`

**Comment by sashachabin** [comment]
> @Omelekhin: we need to move the text below

telegram-cloud-photo-size-2-5195173376338018754-y

**Comment by Omelekhin** [comment]
I've added margin-top for main text and I've made "Find out what we're going to do" button larger and prettier.

File: view/header/index.css [hunk 4]
```diff
@@ -14,33 +14,36 @@
...
   text-decoration: none;
-  transition: .1s ease;
+  box-shadow: 0 .2vw 0 rgba(255,212,0,.5);
+  transition: all 2s cubic-bezier(0.52,0.51,0.39,0.99);
 
   @media (min-width: 768px) {
-    font-size: 3vw;
-    margin: 0 0 -.1em;
+    font-size: 2.5vw;
+		font-weight: 400;
```
Comment (gcor) [inline view/header/index.css]: I see extra spacing here, in file. Maybe Atom can help you autofix on save https://www.google.ru/search?ie=UTF-8&q=autoformat%20atom

File: view/header/index.css [hunk 5]
```diff
@@ -14,33 +14,36 @@
...
+		box-shadow: 0 .25vw 0 rgba(255,212,0,.5);
 
     &:hover {
-      letter-spacing: .2em;
-      margin: 0 -.1em -.1em;
+      /* padding: .5em 1vw .5em 2.5vw; */
```
Comment (gcor) [inline view/header/index.css]: please delete the comment if it is not necessary

**Review by gcor — APPROVED** [review]

**Pull request merged**